### PR TITLE
Links to readme API doc

### PR DESCRIPTION
**Pull request by rnovikovP**
To make users aware of the new API doc (URL TBD, changed later), let's add some linking there to make it more visible. Please, change the context. I'm just highlighting the idea.

### CODE REVIEW

**Comment by rasika-chivate** [comment]
But is not Generally Available right?

File: docs/details/api.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # API
 
+## Interactive API documentation 
+To integrate your applications or CI/CD, you can use our online interactive documentation located at 
+
+https://percona-pmm.readme.io/reference/introduction
+
+## Swagger API documentation
```
Comment (nastena1606) [inline docs/details/api.md:8]: If the docs at readme.io are up-to-date (API methods, I mean), do we need referencing to Swagger at all?

Comment (alexfiru) [reply docs/details/api.md:8]: As far as I understood from our last meeting, they're not up to date. Roma should confirm.

**Review by catalinaadam — CHANGES_REQUESTED** [review]

File: docs/details/api.md (hunk 1 above)
Comment (rnovikovP) [reply docs/details/api.md:7]: @nastena1606, Swagger is still suitable for reference as Local API doc + backup if readme.io is not accurate or not up to date.

@alexfiru, if it's not up to date - it's still a valuable part of the information for users. There are a lot of usage examples.

**Review by denisok — APPROVED** [review]

**Review by nastena1606 — APPROVED** [review]

**Comment by denisok** [comment]
@catalinaadam please see if your feedback was addressed and let's merge

**Review by catalinaadam — APPROVED** [review]

**Pull request merged**